jsdoc/api_ApiBuild.js.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
```

Comment:
this is a result from generating jsdoc, no need to be in git, only for version tagged or ops in general. jsdocs reflect stable ops version not ongoing unstable 'dev'.